packages/core/http/core-http-browser/src/types.ts

```diff
@@ -96,6 +102,12 @@ export interface IBasePath {
    */
   readonly serverBasePath: string;
 
+  /**
+   * Href (hypertext reference) intended to be used as the base for constructing
+   * other hrefs to static assets.
+   */
+  readonly assetsHrefBase: string;
```

Comment:
I added `assetsHrefBase` to `IBasePath`. It wasn't strictly necessary, but I think it makes sense given we're already exposing the `serverBasePath`.